src/components/Book.css

```diff
@@ -0,0 +1,94 @@
+.list-container {
```

Comment:
- [ ] Please there should be a line between the update progress button and the progress bar as in the design 
Zeplin
![image](https://user-images.githubusercontent.com/74744787/222929054-0f12771f-0f82-4064-882d-424276219b89.png)

Your design 
![image](https://user-images.githubusercontent.com/74744787/222929036-975139c5-6448-4e65-8977-c541a3dcc7b4.png)